playground-common/playground-include-settings.gradle

```diff
@@ -64,10 +64,16 @@ def setupPlayground(settings, String relativePathToRoot) {
         }
     }
     settings.includeBuild(new File(supportRoot, "androidx-plugin"))
+    settings.includeProject(":annotation:annotation-sampled",
+            new File(supportRoot, "annotation/annotation-sampled"))
     settings.includeProject(":lint-checks", new File(supportRoot, "lint-checks"))
     settings.includeProject(":fakeannotations", new File(supportRoot,"fakeannotations"))
     settings.includeProject(":internal-testutils-common",
             new File(supportRoot, "testutils/testutils-common"))
+    settings.includeProject(":internal-testutils-ktx",
+            new File(supportRoot, "testutils/testutils-ktx"))
+    settings.includeProject(":internal-testutils-paging",
+            new File(supportRoot, "testutils/testutils-paging"))
```

Comment:
why is paging necessary for all, seems like these should go into the settings of paging.

Reply:
I didn't realize that works. We should probably just declare all of these explicitly in each project to minimize modules loaded